src/lightning_lite/connector.py

```diff
@@ -113,8 +112,8 @@ def __init__(
         # For devices: Assign gpus, ipus, etc. to the accelerator flag and devices flag
         self._strategy_flag: Optional[Union[Strategy, str]] = None
         self._accelerator_flag: Optional[Union[Accelerator, str]] = None
-        self._precision_flag: Optional[_PRECISION_INPUT] = None
-        self._precision_plugin_flag: Optional[Precision] = None
+        self._precision_input: Optional[_PRECISION_INPUT] = None
+        self._precision_instance: Optional[Precision] = None
```

Comment:
I wouldn't diverge from the accelerator connector class unless absolutely necessary. When we need to make changes, or fixes, we likely need to update both classes (today), and inconsistencies will make it only more difficult.

Reply:
I think they will inevitably diverge if we are not going to port API changes to the Trainer and its flags that we will do to Lite. This will be a challenge in the coming months, but we don't know what will be the end result. For this reason, I wouldn't block changes to Lite just because we don't know what will happen in PL.

Reply:
I would not hold back on anything that simplifies or does meaningful refactors. But simple name changes on internal flags just makes it harder when it doesn't have to be.